Makefile

```diff
@@ -0,0 +1,4 @@
+test:
+	cmake --build build
```

Comment:
This fails on a fresh install. I need to run `cmake -B build` first. Can we separate both of these into a build action that test depends on?

Reply:
i'll fix this on the other branch. merging this for now